frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/status-card/status-card.tsx

```diff
@@ -0,0 +1,113 @@
+import AlertsBody from '@console/shared/src/components/dashboard/status-card/AlertsBody';
+import AlertItem from '@console/shared/src/components/dashboard/status-card/AlertItem';
+import { alertURL, PrometheusRulesResponse } from '@console/internal/components/monitoring';
+import DashboardCard from '@console/shared/src/components/dashboard/dashboard-card/DashboardCard';
+import DashboardCardBody from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardBody';
+import DashboardCardHeader from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardHeader';
+import DashboardCardTitle from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardTitle';
+import HealthBody from '@console/shared/src/components/dashboard/status-card/HealthBody';
+import HealthItem from '@console/shared/src/components/dashboard/status-card/HealthItem';
+import { getAlerts } from '@console/shared/src/components/dashboard/health-card/utils';
+import { PrometheusResponse } from '@console/internal/components/graphs';
+import {
+  withDashboardResources,
+  DashboardItemProps,
+} from '@console/internal/components/dashboard/with-dashboard-resources';
+import {
+  DATA_RESILIENCY_QUERY,
+  STORAGE_HEALTH_QUERIES,
+  StorageDashboardQuery,
+} from '../../../../constants/queries';
+import { filterCephAlerts } from '../../../../selectors';
+import { getCephHealthState, getDataResiliencyState } from './utils';
+
+const CephAlerts = withDashboardResources(({ watchAlerts, stopWatchAlerts, alertsResults }) => {
+  React.useEffect(() => {
+    watchAlerts();
+    return () => {
+      stopWatchAlerts();
+    };
+  }, [watchAlerts, stopWatchAlerts]);
+
+  const alertsResponse = alertsResults.getIn([ALERTS_KEY, 'data']) as PrometheusRulesResponse;
+  const alertsResponseError = alertsResults.getIn([ALERTS_KEY, 'loadError']);
+  const alerts = filterCephAlerts(getAlerts(alertsResponse));
+
+  return (
+    <AlertsBody
+      isLoading={!alertsResponse}
+      error={alertsResponseError}
+      emptyMessage="No persistent storage alerts"
+    >
+      {alerts.length &&
+        alerts.map((alert) => <AlertItem key={alertURL(alert, alert.rule.id)} alert={alert} />)}
+    </AlertsBody>
+  );
+});
+
+export const StatusCard: React.FC<DashboardItemProps> = ({
+  watchK8sResource,
+  stopWatchK8sResource,
+  watchPrometheus,
+  stopWatchPrometheusQuery,
+  prometheusResults,
+}) => {
+  React.useEffect(() => {
+    watchPrometheus(STORAGE_HEALTH_QUERIES[StorageDashboardQuery.CEPH_STATUS_QUERY]);
+    watchPrometheus(DATA_RESILIENCY_QUERY[StorageDashboardQuery.RESILIENCY_PROGRESS]);
+
+    return () => {
+      stopWatchPrometheusQuery(STORAGE_HEALTH_QUERIES[StorageDashboardQuery.CEPH_STATUS_QUERY]);
+      stopWatchPrometheusQuery(DATA_RESILIENCY_QUERY[StorageDashboardQuery.RESILIENCY_PROGRESS]);
+    };
+  }, [watchK8sResource, stopWatchK8sResource, watchPrometheus, stopWatchPrometheusQuery]);
```

Comment:
As mentioned [before](https://github.com/openshift/console/pull/2897#discussion_r332021838), do we really need `watchK8sResource` and  `stopWatchK8sResource` props?

They are not used within the `StatusCard` component, so we can omit them from props destructuring as well as `React.useEffect` dependencies.